docker-entrypoint.sh

```diff
@@ -11,13 +11,19 @@ if test $# -eq 0; then
     exit 1
 fi
 
-fabmanager list-users --app superset # Run this to report any config errors visibly. https://github.com/dpgaspar/Flask-AppBuilder/issues/742
-user_count="$(fabmanager list-users --app superset | sed -n '/^username/p' | wc -l)"
-if test "${user_count}" = 0; then
-    stderr "No users exist. Creating admin user."
-    fabmanager create-admin --app superset --username admin --password admin \
-        --firstname Admin --lastname Admin --email admin@localhost
+SUPERSET_ADMIN_USERNAME="${SUPERSET_ADMIN_USERNAME-admin}"
+SUPERSET_ADMIN_PASSWORD="${SUPERSET_ADMIN_PASSWORD:-admin}"
+if test ! -z "${SUPERSET_ADMIN_USERNAME}"; then
```

Comment:
Can `SUPERSET_ADMIN_USERNAME` be unset? Can it be empty? Maybe fail if it is empty.

Reply:
I consciously allow `SUPERSET_ADMIN_USERNAME` be empty. In that case, no user is created.